packages/clerk-js/src/ui/router/__tests__/PathRouter.test.tsx

```diff
@@ -54,6 +54,7 @@ describe('PathRouter', () => {
   beforeAll(() => {
     // @ts-ignore
     delete window.location;
+    history.replaceState = jest.fn();
```

Comment:
🙃 could you please add an assertion that it's being called with the expected values?